mesheryctl/cmd/perf.go

```diff
@@ -123,14 +166,13 @@ var perfCmd = &cobra.Command{
 }
 
 func init() {
-	perfCmd.Flags().StringVar(&testURL, "url", "", "(required) URL of the endpoint to use for the test")
-	perfCmd.Flags().StringVar(&testName, "name", StringWithCharset(8), "(optional) A memorable name for the test.")
-	perfCmd.Flags().StringVar(&testMesh, "mesh", "", "(optional) Name of the service mesh.")
-	// perfCmd.Flags().StringVar(&testFile, "file", "", "DESCRIPTION")
+	perfCmd.Flags().StringVar(&testURL, "url", "", "(required) Endpoint URL to test")
+	perfCmd.Flags().StringVar(&testName, "name", "", "(optional) Name of the Test")
+	perfCmd.Flags().StringVar(&testMesh, "mesh", "", "(optional) Name of the Service Mesh")
 	perfCmd.Flags().StringVar(&qps, "qps", "0", "(optional) Queries per second")
-	perfCmd.Flags().StringVar(&concurrentRequests, "concurrent-requests", "1", "DESCRIPTION")
-	perfCmd.Flags().StringVar(&testDuration, "duration", "30s", "(optional) Duration of the test like 10s, 5m, 2h. We are following the convention described at https://golang.org/pkg/time/#ParseDuration")
-	perfCmd.Flags().StringVar(&testCookie, "cookie", "meshery-provider=Default Local Provider", "(required) identification of choice of provider.")
-	perfCmd.Flags().StringVar(&loadGenerator, "load-generator", "fortio", "	(optional) choice of load generator: fortio (OR) wrk2")
+	perfCmd.Flags().StringVar(&concurrentRequests, "concurrent-requests", "1", "(required) Number of Parallel Requests")
+	perfCmd.Flags().StringVar(&testDuration, "duration", "30s", "(optional) Duration of the Test")
```

Comment:
If we can make the original description more concise, that would be good. Let's see if we can continue to provide users with helpful, specific examples of other uses (5m, 2h). Perhaps, a shorter description that retains its helpfulness is:

```
(optional) Length of test (e.g. 10s, 5m, 2h). For more, see https://golang.org/pkg/time/#ParseDuration"
```

It would be good to avoid use of the word "Duration" in the description given that this word is used in the parameter name. Eventually, we should replace the URL to Golang docs with a URL to Meshery docs.

Reply:
Done !